Source/Fuse.Maps/MapView.uno

```diff
@@ -219,6 +240,16 @@ namespace Fuse.Controls
 			UpdateMarkersNextFrame();
 		}
 
+		void OnPolylineAdded(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
+
+		void OnPolylineRemoved(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
```

Comment:
I agree, but I thought there might be a reason why you had it in two different:

https://github.com/fusetools/fuselibs-public/blob/master/Source/Fuse.Maps/MapView.uno#L212-L220

		void OnMarkerAdded(MapMarker marker)
		{
			UpdateMarkersNextFrame();
		}

		void OnMarkerRemoved(MapMarker marker)
		{
			UpdateMarkersNextFrame();
		}